File: V2EX/AllTopic/TopicsTVC/topicDetail/RLReplyCell.m
//
//  RLReplyCell.m
//  V2EX
//
//  Created by rylynn lai on 16/3/12.
//  Copyright © 2016年 ucs. All rights reserved.
//

#import "RLReplyCell.h"
#import "RLTopicReply.h"

@interface RLReplyCell ()
@property (weak, nonatomic) IBOutlet UIImageView *authorIcon;
@property (weak, nonatomic) IBOutlet UILabel *contentLable;
@property (weak, nonatomic) IBOutlet UILabel *createdTimeAndThanks;
@property (weak, nonatomic) IBOutlet UIButton *authorBtn;
@end

@implementation RLReplyCell

- (void)setReply:(RLTopicReply *)reply {
    _reply = reply;
}

- (void)awakeFromNib {
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: V2EX/AllTopic/TopicsTVC/topicDetail/RLReplyCell.swift
//
//  RLReplyCell.swift
//  V2EX
//
//  Created by LLZ on 16/3/28.
//  Copyright © 2016年 ucs. All rights reserved.
//

import UIKit

class RLReplyCell: UITableViewCell {
    
    var reply:RLTopicReply? 
    
    @IBOutlet weak var indexLable: UILabel!
    
    @IBOutlet weak var authorIcon: UIImageView!
    
    @IBOutlet weak var contentLable: UILabel!
    
    @IBOutlet weak var createdTimeAndThanks: UILabel!
    
    @IBOutlet weak var authorBtn: UIButton!
    
    
    
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
